docs/docs/pyproject.md

```diff
@@ -212,6 +212,22 @@ poetry = 'poetry.console:run'
 
 Here, we will have the `poetry` script installed which will execute `console.run` in the `poetry` package.
 
+### `script-files`
+
+You can also include non-Python scripts in the distribution package.
+
+```toml
+[tool.poetry]
+script-files = ["contrib/run_db_migration.sh", "bin/healthcheck.py"]
```

Comment:
The @python-poetry/core team has decided on going forward with keeping the `[tool.poetry.scripts]` namespace and using inline tables to have more flexibility and backwards compatibility.